homeassistant/components/gtfs/sensor.py

```diff
@@ -490,7 +490,7 @@ def setup_platform(
     origin = config.get(CONF_ORIGIN)
     destination = config.get(CONF_DESTINATION)
     name = config.get(CONF_NAME)
-    offset = config.get(CONF_OFFSET)
+    offset = datetime.timedelta(seconds=float(config.get(CONF_OFFSET, 0)))
```

Comment:
This is a breaking change and should be reverted.
This value could previously be a number that was mapped to seconds or a `time_period_str` (and even a `time_period_dict`) that would get a proper timedelta

Reply:
Oh shit, you are right, I've locally checked this out even and checked the flow, but missed the config schema.